src/modules/collection/sagas.spec.ts

```diff
@@ -2010,3 +2013,236 @@ describe('when handling the fetch of a collection', () => {
     })
   })
 })
+
+describe.skip('when publishing a collection with fiat', () => {
```

Comment:
We can skip the successful tests, the failing ones can still be kept, as their sagas will finish.